src/components/AppNavigation/AppNavigation.vue

```diff
@@ -188,4 +188,11 @@ export default {
 	}
 }
 
+// Put the toggle behind appsidebar on small screens
+@media only screen and (max-width: math.div($breakpoint-mobile, 2)) {
+	.app-navigation {
+		z-index: 1400;
```

Comment:
@raimund-schluessler we could alternatively increase the z-index of the app-sidebar to above 1800 which would do the same but we would have everything in one place. WDYT? :)

Reply:
I have no preference here. Both seems fine.